folding-schemes/src/transcript/poseidon/grumpkin.rs

```diff
@@ -0,0 +1,518 @@
+use crate::utils::vec::mat_from_str_mat;
+use ark_crypto_primitives::sponge::poseidon::PoseidonConfig;
+use ark_ff::{BigInteger, PrimeField};
+use ark_grumpkin::Fr;
+
+// Poseidon constants for Grumpkin curve as generated by
+// https://extgit.iaik.tugraz.at/krypto/hadeshash/-/blob/master/code/generate_params_poseidon.sage
```

Comment:
Proposing to also add sage script's arguments to ease reproducibility. Checked on my side and correct me if I'm wrong, but should be those ones?

```suggestion
// https://extgit.iaik.tugraz.at/krypto/hadeshash/-/blob/master/code/generate_params_poseidon.sage
// generated with `sage generate_params_poseidon.sage 1 0 254 5 5 128 0x30644e72e1
31a029b85045b68181585d97816a916871ca8d3c208c16d87cfd47`
```